src/shell/element/mod.rs

```diff
@@ -759,6 +759,22 @@ impl CosmicMapped {
             popup_elements.into_iter().map(C::from).collect(),
         )
     }
+
+    pub(crate) fn set_theme(&self) {
```

Comment:
Since this has no parameter, could we maybe rename this to something like `reload_theme` or `update_theme`? (Same then for `CosmicStack`, `CosmicWindow`, `IcedElement`..)